fvm/evm/emulator/state/stateDB.go

```diff
@@ -481,6 +506,10 @@ func (db *StateDB) Commit(finalize bool) (hash.Hash, error) {
 	return updateCommit, nil
 }
 
+func (db *StateDB) Finalise(deleteEmptyObjects bool) {
+	panic(fmt.Errorf("should not be called!"))
```

Comment:
I wonder if it would be better to call `Finalise` but leave it empty. Might be more in line with how its envisioned.

Reply:
Good point indeed. Updated in https://github.com/onflow/flow-go/pull/7241/commits/a69d18c199b3dd7184120dc50fc8d778e46d6721 .